src/Uno.UI/UI/Xaml/Input/PointerRoutedEventArgs.Android.cs

```diff
@@ -15,17 +15,29 @@ namespace Windows.UI.Xaml.Input
 {
 	partial class PointerRoutedEventArgs
 	{
+		private const int _pointerIdsCount = (int)MotionEventActions.PointerIndexMask >> (int)MotionEventActions.PointerIndexShift; // 0xff
+		private const int _pointerIdsShift = 31 - (int)MotionEventActions.PointerIndexShift; // 23
```

Comment:
You should document those magic numbers

Reply:
It's documented where I use those values (cf. line 31)